less/_grid.less

```diff
@@ -188,11 +188,12 @@
 	}
 	// when the page has a max-width set
 	&.max-width {
-		@max-width: 1280px;
-		max-width: @max-width;
+		max-width: @pg-width-max;
 		margin: auto;
-		@media screen and (min-width: unit(@mediaquery-sm, px)) and (max-width: @max-width - 1) {
-
+		> [class*='col'] {
```

Comment:
removes extra padding from columns directly inside the `.grid.max-width` wrapper :balloon: